assets/stylesheets/_variables.scss

```diff
@@ -9,6 +9,7 @@ $default-line-height: 1.4;
 $editor-font: "Noto Serif", serif;
 $editor-html-font: Menlo, Consolas, monaco, monospace;
 $editor-font-size: 16px;
+$default-block-margin: 28px; // Why not ems? Because 32px provides a consistent block spacing based on the 16px font size.
```

Comment:
Super-minor, but can we edit or remove the code comment here? It doesn't quite make sense now that the value is `28px`, not `32px`.